app/controllers/publish/courses/subjects_controller.rb

```diff
@@ -16,8 +16,19 @@ def continue
 
       def update
         authorize(provider)
+        if params[:course][:master_subject_id] == "29" && FeatureService.enabled?(:engineers_teach_physics_on_course)
```

Comment:
I think this should use `SecondarySubject.physics.id`?

Reply:
Yep, to_s